tests/savings.test.js

```diff
@@ -0,0 +1,66 @@
+import HomePage from "../pages/HomePage";
+import SignIn from "../pages/SignIn";
+import SavingsForm from "../pages/SavingsForm"
+import SavingsTable from "../pages/SavingsTable";
+import CorrectSavings from "../datos/CorrectSavings";
+import IncorrectCheckings from "../datos/IncorrectSavings"
+
+
+beforeEach(async () => {
```

Comment:
This should be implemented inside the `describe` callback, at the same level than the rest of the tests